Composer/packages/client/src/pages/design/VisualEditor.tsx

```diff
@@ -27,11 +27,16 @@ const addIconProps = {
   styles: { root: { fontSize: '12px' } },
 };
 
-function onRenderBlankVisual(isTriggerEmpty, onClickAddTrigger) {
+function onRenderBlankVisual(isTriggerEmpty, onClickAddTrigger, isRemoteSkill) {
   return (
     <div css={middleTriggerContainer}>
       <div css={middleTriggerElements}>
-        {isTriggerEmpty ? (
+        {isRemoteSkill ? (
+          <React.Fragment>
+            <img alt={formatMessage('bot framework composer icon gray')} src={grayComposerIcon} />
```

Comment:
I think just `Composer icon` is enough for the alt-text here.

Reply:
That's the previous existed alt-text, when no dialog/trigger selected, can you confirm it with design team?

Reply:
Okay, if that's what was there before then let's keep it. I'll make a note to talk to the design team later.